examples/raytracing-rahit/shader.rahit

```diff
@@ -0,0 +1,20 @@
+#version 460 core
+#extension GL_EXT_ray_tracing : require
+
+layout(location = 0) in float opacity; // 0.0f = transparent, 1.0f = opaque
+
+void main()
+{
+    // The very first opaque geometry evaluated will terminate the ray
+    // Round about way so that interpreter can work with these
```

Comment:
I don't understand this comment. What does the shader do differently to cooperate with the interpreter?

Reply:
GLSL clamp instruction wasn't implemented in the interpreter yet, so I placed that comment. Looking at it again, it looks confusing without context so I will remove it.